src/components/List/ListActions/ListActions.tsx

```diff
@@ -1,11 +1,15 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import bem from '../../../utils/bem';
 import styles from './ListActions.scss';
 
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Actions to be pushed to the left side of a List Item */
+    children?: ReactNode;
+}
+
 const { block } = bem('ListActions', styles);
 
-const ListActions = props => {
+const ListActions: React.FC<Props> = props => {
     const { children, ...rest } = props;
```

Comment:
Shouldn't we add default property for `children` here as long as we removed it from `defaultProps`?

Like:
```
const { children = null, ...rest } = props;
```

Reply:
Just as a side note, the whole component can be more "compact".

```tsx

const ListActions: React.FC<Props> = ({ children, ...rest }) => (
    <div {...rest} {...block(rest)}>
        {children}
    </div>
);
```

Reply:
What would be the reason to define children as `null` instead of undefined?

Reply:
Yeah, should be  `undefined` :)